app/controllers/api/school_search_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Api::SchoolSearchController < Api::ApiController
+  def index
```

Comment:
@DFE-Digital/ecf Does this new feature mean we can remove https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/controllers/school_search_controller.rb

Reply:
Almost certainly not. We will still need `school_search_controller` (or any other controller) to display the school search page. But we can probably prune away the search logic from there, with enough effort. I would do it in a separate PR though.

Reply:
No - this is only for school search api access

Reply:
It's namespaced so it doesn't conflict